src/test/java/io/appform/dropwizard/discovery/bundle/ServiceDiscoveryBundleTest.java

```diff
@@ -30,6 +33,7 @@
 import io.dropwizard.setup.AdminEnvironment;
 import io.dropwizard.setup.Bootstrap;
 import io.dropwizard.setup.Environment;
+import jdk.nashorn.internal.runtime.regexp.joni.Config;
```

Comment:
Nashorn is deprecated

Reply:
Must be during autocomplete, not used anywhere, removed